sys/shell/commands/Makefile

```diff
@@ -44,12 +38,11 @@ endif
 ifneq (,$(filter gnrc_ipv6_blacklist,$(USEMODULE)))
   SRC += sc_blacklist.c
 endif
-# The ping command in sc_icmpv6_echo requires xtimer, too. However, this is
-# implicitly pulled in by gnrc_ipv6_netif (which is a dependency of gnrc_ipv6)
-# already.
 ifneq (,$(filter gnrc_icmpv6_echo,$(USEMODULE)))
+ifneq (,$(filter xtimer,$(USEMODULE)))
```

Comment:
Why the explicit check for xtimer?

Reply:
See the removed comment above. `gnrc_netif2` doesn't have this dependency. `gnrc_ipv6_nib` has it, but less obvious and I also think that dependencies should be as explicit as possible.

Reply:
So only the shell command requires the xtimer (and not the icmpv6 echo code itself). From an end user perspective, I would be suprised not to get the shell command when enabling the `gnrc_icmpv6_echo` module. I don't think the explicit check here is bad, I'm mostly doubting whether instead of an `ifneq` check, just pulling in xtimer as an additional dependency makes more sense.

As an example, on this branch, the gnrc_networking example includes xtimer due to the inclusing of RPL, if say a user were to remove the RPL he suddenly loses the icmpv6 command for no immediatly visible reason.

Reply:
> So only the shell command requires the xtimer (and not the icmpv6 echo code itself). From an end user perspective, I would be suprised not to get the shell command when enabling the gnrc_icmpv6_echo module. I don't think the explicit check here is bad, I'm mostly doubting whether instead of an ifneq check, just pulling in xtimer as an additional dependency makes more sense.

It would be very inconsistent to pull it in here, also: other shell commands do it like I do here as well (and this PR is not about changing that).

> As an example, on this branch, the gnrc_networking example includes xtimer due to the inclusing of RPL, if say a user were to remove the RPL module, he suddenly loses the icmpv6 command for no immediatly visible reason.

Then still `gnrc_ipv6_nib` would (indirectly) include `xtimer`. It is very unlikely that a user using `gnrc_ipv6` would not use `xtimer` as well. Anyone who does I assume knows what they are doing ;-).

Reply:
> It would be very inconsistent to pull it in here, also: other shell commands do it like I do here as well (and this PR is not about changing that).

Okay, I can live with this :)

> Anyone who does I assume knows what they are doing ;-).

Lets hope so :)